### DISCUSSION TITLE
My Chia.net will not connect yesterday or today?

### DISCUSSION THREAD

**Discussion by CWR009** [Chia-Network/chia-blockchain#6684] [Support]
I have not changed anything but Webroot came up with a Lazurus virus on one of my SSD on my Mac.  I removed it but now It is continually trying to connect to Chia but does not.

**Comment by loppefaaret** [comment]
Do you mean you open up the Chia app, and all you are seeing is a "Connecting to wallet..." message ?

**Comment by BrandtH22** [comment]
@CWR009 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
